_posts/2015-08-25-almundo-bill-vulnerability.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: "Almundo's Bill Vulnerability"
-date: 2015-08-25 03:55:00
+date: 2015-03-25 03:55:00
```

Comment:
@ABizzinotto you missed the reviewed date here

Reply:
Added